website/views.py

```diff
@@ -120,7 +120,7 @@ def index():
 
 def find_bookmark_collection(user):
     Collection = apps.get_model('osf.Collection')
-    return Collection.objects.get(creator=user, is_deleted=False)
+    return Collection.objects.get(creator=user, is_deleted=False, is_bookmark_collection=True)
```

Comment:
This won't ever fail to return, right? Because you'll get an exception if it does. But from the diff, you would always.